examples/search_examples/public/search_sessions/app.tsx

```diff
@@ -690,9 +690,8 @@ function doSearch(
   const startTs = performance.now();
 
   // Submit the search request using the `data.search` service.
-  return data.search
-    .search(req, { sessionId })
-    .pipe(
+  return firstValueFrom(
```

Comment:
I think this should be `lastValueFrom` ?

Reply:
good catch! done!